src/map/battle.cpp

```diff
@@ -9628,9 +9628,7 @@ struct Damage battle_calc_misc_attack(struct block_list *src,struct block_list *
 			break;
 #ifndef RENEWAL
 		case BA_DISSONANCE:
-			md.damage = 30 + skill_lv * 10;
-			if (sd)
-				md.damage += 3 * pc_checkskill(sd,BA_MUSICALLESSON);
+			md.damage = 30 + skill_lv * (10 + pc_checkskill(sd, BA_MUSICALLESSON));
```

Comment:
```suggestion
			md.damage = 30 + skill_lv * 10;
			md.damage += pc_checkskill(sd, BA_MUSICALLESSON) * skill_lv;
```

Reply:
Why? Isn't the other syntax much nicer?

Reply:
Most skill descriptions have those parts split up like this.
So it is much easier to compare the formula you are reading to the skill description.
You can also see that with the renewal description: [Divine Pride](https://divine-pride.net/database/skill/317/unchained-serenade)
Here the MATK part is your first line, whereas the second line would be something like "Damage increases as the caster's job level increases." or even inside another (passive) skills description.

Additionally it enables us to track changes to specific parts of a skill formula that has been changed.

Also it resolves the issue, that each developer has a different way to show how good he/she is in math and how much he/she can shorten and simplify the formula.

Reply:
Well I can update it, but note that this is the pre-renewal formula, I'm still working on the renewal one.
For pre-renewal people would consult Kokotewa: https://ro.kokotewa.com/db/skl_info?id=BA_DISSONANCE
That's why I designed it the way Koko wrote it.